src/apps/dashboard/dashboard.dev.tsx

```diff
@@ -134,6 +134,10 @@ export default {
       },
       defaultValue: "Number @count in line"
     },
+    reservationDetailsOthersInQueueText: {
+      defaultValue: "Others are queueing for this material",
```

Comment:
Have you double checked our code for a similar text? Wouldn't it make sense to reuse translations in some places instead of having different context everywhere?